index.d.ts

```diff
@@ -11,5 +11,6 @@ normalizeNewline(Buffer.from('foo\r\nbar\nbaz')).toString();
 //=> 'foo\nbar\nbaz'
 ```
 */
-export function normalizeNewline(input: string): string;
-export function normalizeNewline(input: Buffer): Buffer;
+declare function normalizeNewline(input: string): string;
+declare function normalizeNewline(input: Buffer): Buffer;
+export default normalizeNewline;
```

Comment:
Still incorrect. No need for `declare`.

Reply:
Hum... but with no 'declare' I get `Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.`

So if I am incorrect, I am likely clueless too :/

Reply:
You don't fancy this one-liner, do you?
```ts
export default function normalizeNewline<Input extends string | Buffer>(input: Input): Input;
```